src/gui/sdlmain.cpp

```diff
@@ -4476,8 +4221,7 @@ void config_add_sdl() {
 	pstring->Set_values(presentation_modes);
 
 	const char* outputs[] =
-	{ "surface",
-	  "texture",
+	{ "texture",
```

Comment:
Please update `windowresolution` help string, it mentions `output=surface`:

```C++
	pstring = sdl_sec->Add_string("windowresolution", on_start, "default");
	pstring->Set_help(
	        "Set window size when running in windowed mode:\n"
	        "  default:   Select the best option based on your environment and\n"
	        "             other settings.\n"
	        "  small, medium, large (s, m, l):\n"
	        "             Size the window relative to the desktop.\n"
	        "  <custom>:  Scale the window to the given dimensions in WxH format.\n"
	        "             For example: 1024x768.\n"
	        "             Scaling is not performed for output=surface.");
```

Reply:
`git grep -i surface`

Reply:
Backported from @kcgen's branch.